docs/tutorials/materialized.md

```diff
@@ -310,18 +310,17 @@ CREATE STREAM calls WITH (
 
 A common situation in call centers is the need to know what the current caller has called about in the past. Create a simple materialized view that keeps track of the distinct number of reasons that a user called for, and what the last reason was that they called for, too. This gives you an idea of how many kinds of inquiries the caller has raised and also gives you context based on the last time they called.
 
-You do this by declaring a table called `support_view`. Keeping track of the distinct number of reasons a caller raised is as simple as grouping by the user name, then aggregating with `count_distinct` over the `reason` value. Similarly, you can retain the last reason the person called for with the `latest_by_offset` aggregation.
```

Comment:
This example should be change starting further above
```
### Create the ksqlDB calls stream
For ksqlDB to be able to use the topic that Debezium created, you must declare a stream over it.
```

We don't need to create a STREAM first any longer, but the two statement `CREATE STREAM` plus `CREATE TABLE AS SELECT` are replace by a single `CREATE SOURCE TABLE <schema> WITH(...)` statement.

We need to tell the whole story a little differently.

Reply:
Cool, can you suggest an edit, @MichaelDrogalis?